core/schema/coremod.go

```diff
@@ -13,39 +13,64 @@ import (
 	"github.com/dagger/dagger/dagql/idproto"
 )
 
+// the core mod served to all clients, including modules
+func FixedCoreMod(dag *dagql.Server) *CoreMod {
```

Comment:
nit: what does "Fixed" mean? :thinking: 

...but I dunno what else I'd call it. (CoreCore? Nucleus? CoreWithoutHost?)

Reply:
Yeah it is not a great name 😂 The overly verbose name would be `CoreModThatIsServedToAllClientsWhetherOrNotTheyAreAModule`, so I went with "fixed" in that they are fixed to always be served to any client, but that barely makes sense. 

I'll just update it to `CoreWithoutHost` for now, that's one increment better than `FixedCoreMod` I think 👍